### PR TITLE
BREAKING: Adapt to latest module template standards

### PR DESCRIPTION
**Pull request by Mrtenz**
This adapts the package to the latest module template standards.

- Updated Node to 14.
- Updated Yarn to 3.
- Updated all workflows, configs, etc. to match the module template.
- Bumped some dependencies.

### CODE REVIEW

File: .github/workflows/fetch-and-rebuild.yml [hunk 1]
```diff
@@ -12,29 +12,19 @@ jobs:
...
-      - run: yarn allow-scripts
-
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+      - name: Install Yarn dependencies
+        run: yarn --immutable
       - name: Fetch, rebuild, create pull request
         run: yarn rebuild:ci ${{ steps.date.outputs.date }}
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (FrederikBolding) [inline .github/workflows/fetch-and-rebuild.yml:40]: Why is this being changed?

**Comment by FrederikBolding** [comment]
Also shouldn't this be marked as breaking?

File: .github/workflows/fetch-and-rebuild.yml (hunk 1 above)
Comment (Mrtenz) [reply .github/workflows/fetch-and-rebuild.yml:40]: It just seemed clearer to name it `GITHUB_TOKEN`.

Comment (FrederikBolding) [reply .github/workflows/fetch-and-rebuild.yml:40]: I can't find where it is used though, so I'm not sure if it'll break 😅

Comment (Mrtenz) [reply .github/workflows/fetch-and-rebuild.yml:40]: GitHub CLI. It accepts both `GH_TOKEN` and `GITHUB_TOKEN`.

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**